src/styles/blocks/form-field.scss

```diff
@@ -0,0 +1,31 @@
+.form-field {
+  font-family: "Cinzel", sans-serif;
+  box-sizing: border-box;
+  display: block;
+  width: 100%;
+  height: 48px;
+  background-color: #f8f8fa;
+  border: 1px solid #d5d7ed;
+
+  &::placeholder {
```

Comment:
Add ```
content: '';```, 
otherwise, the ::placeholder pseudo-element may not work as expected in some browsers.